src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpOnFilenameCheck.java

```diff
@@ -101,6 +101,13 @@
  * <pre>
  * &lt;module name=&quot;RegexpOnFilename&quot;/&gt;
  * </pre>
+ *
+ * <p>Example:</p>
+ * <pre>
+ * src/xdocs/config_regexp.xml  //OK, contains no whitespace
+ * src/xdocs/&quot;config regexp&quot;.xml  //violation, contains whitespace
+ * </pre>
+ *
  * <p>
  * To configure the check to force picture files to not be 'gif':
```

Comment:
>  to force picture files to not be 'gif'

please do me a favor and change it to ` to forbid 'gif' files in folders`

Reply:
Done